mlir/test/Conversion/VectorToArmSME/vector-to-arm-sme.mlir

```diff
@@ -718,18 +718,6 @@ func.func @vector_load_i8_with_offset(%arg0 : memref<?x?xi8>) -> vector<[16]x[16
 
 // -----
 
-// CHECK-LABEL: @vector_load_i8_from_rank_1_memref(
-// CHECK-SAME:                                     %[[MEMREF:.*]]: memref<?xi8>)
-// CHECK: %[[C0:.*]] = arith.constant 0 : index
-// CHECK: arm_sme.tile_load %[[MEMREF]][%[[C0]]] : memref<?xi8>, vector<[16]x[16]xi8>
```

Comment:
You could follow up to apply the same changes to ArmSME. There's some unused code `ArmSMEToSCF.cpp` for dealing with rank 1 memrefs (e.g. in `getMemrefIndices`) and it somewhat broken: https://github.com/llvm/llvm-project/issues/118769.